packages/regenerator-transform/src/visit.js

```diff
@@ -11,15 +11,14 @@
 "use strict";
 
 import assert from "assert";
-import * as t from "babel-types";
 import { hoist } from "./hoist";
 import { Emitter } from "./emit";
 import replaceShorthandObjectMethod from "./replaceShorthandObjectMethod";
 import * as util from "./util";
 
-exports.visitor = {
+exports.getVisitor = ({ types: t }) => ({
   Function: {
-    exit: function(path, state) {
+    exit: util.wrapWithTypes(t, function(path, state) {
```

Comment:
Everything `regenerator-transform` does (or, more specifically, all code that calls `util.getTypes()`) happens within this `exit` method, so this is the only function that we need to wrap.